defaults/main.yml

```diff
@@ -13,12 +13,14 @@ php_fpm_log_level: error
 # php_fpm_process_priority: -19
 # php_fpm_rlimit_core: 0
 # php_fpm_rlimit_files: 1024
-php_fpm_run_dir: "/var/run"
+php_fpm_run_dir: "/var/run/php"
 # php_fpm_syslog_facility: "daemon"
 # php_fpm_syslog_ident: "php5-fpm"
 # php_fpm_systemd_interval: 10
 php_fpm_user: "www-data"
-
+php_fpm_socket_filename: "php7.0-fpm.sock"
+php_fpm_listen: "/var/run/php/php7.0-fpm.sock"
+php_fpm_listen_allowed_clients: "127.0.0.1"
```

Comment:
Shouldn't this line be commented out by default? Otherwise it will always be set, and always be written to `pool.conf`